### DISCUSSION TITLE
How to run the generated project locally?

### DISCUSSION THREAD

**Discussion by HiddenUndefined** [nuxt/nuxt#18312] [Questions] [answered]
In `Nuxt 2` I use npm package `serve` for run generated project from `dist` folder.

But in `Nuxt 3` `nuxt build` create new folder `.output` contains two folder `server` & `public`.

How to run the generated project locally?

**Comment by HiddenUndefined** [comment, marked as answer]
Found a simple solution in the documentation :)

Run terminal command in root project folder:
```
node .output/server/index.mjs
```

**Reply by HenrijsS** [reply]
Or just `npm run start` or `yarn start`
